### PR TITLE
Postprocess

### PR DESCRIPTION
**Pull request by buehlere**
Adding cwl support for [postprocessing_variant_calls](https://github.com/msk-access/postprocessing_variant_calls/tree/re-project-layout) in cwl-commandlinetools. Currently adding support for: 
- `pv vardict single filter` Tested with singularity and toil on juno: 
    - [singularity_single.txt](https://github.com/mskcc/cwl-commandlinetools/files/8711353/singularity_single.txt)
    - [toil_single.txt](https://github.com/mskcc/cwl-commandlinetools/files/8711355/toil_single.txt)
- `pv vardict case-control filter`. Tested with singularity and toil on juno: 
    - [singularity_case-control.txt](https://github.com/mskcc/cwl-commandlinetools/files/8711352/singularity_case-control.txt)
    - [toil_case-control.txt](https://github.com/mskcc/cwl-commandlinetools/files/8711354/toil_case-control.txt)
  

Let me know how things look!

### CODE REVIEW

File: pv_vardict_0.0.1/case-control_filter_0.0.1.cwl [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  - filter
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 40000
+    coresMin: 4
+  - class: InlineJavascriptRequirement
+hints:
+  - class: DockerRequirement
+    dockerPull: 'ghcr.io/msk-access/postprocessing_variant_calls:latest'
```
Comment (rhshah) [inline pv_vardict_0.0.1/case-control_filter_0.0.1.cwl:95]: We should use a specific version of the release and not the latest and I think we should make sure you make the release of the  postprocessing_variant_calls and then use that tag of the docker image

File: pv_vardict_0.0.1/single_filter_0.0.1.cwl [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  - filter
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 40000
+    coresMin: 4
+  - class: InlineJavascriptRequirement
+hints:
+  - class: DockerRequirement
+    dockerPull: 'ghcr.io/msk-access/postprocessing_variant_calls:latest'
```
Comment (rhshah) [inline pv_vardict_0.0.1/single_filter_0.0.1.cwl:95]: We should use a specific version of the release and not the latest and I think we should make sure you make the release of the  postprocessing_variant_calls and then use that tag of the docker image

**Review by rhshah — CHANGES_REQUESTED** [review]
We might want to call the folder `pv` and then add version `0.0.1` and then the files can be `pv_vardict_filter_case-control.cwl` and `pv_vardict_filter_single-sample.cwl`. Also please add the folder pv with `README.md` with just `# Posprocessing of variant calls` and `vardict_filter_case-control.md` and `vardict_filter_single-sample.md`

File: pv_vardict_0.0.1/case-control_filter_0.0.1.cwl (hunk 1 above)
Comment (buehlere) [reply pv_vardict_0.0.1/case-control_filter_0.0.1.cwl]: fix in this commit: https://github.com/msk-access/cwl-commandlinetools/pull/148/commits/be5e53e5c5b8838222b3b1f477a0785684f4dab0 !

File: pv_vardict_0.0.1/single_filter_0.0.1.cwl (hunk 2 above)
Comment (buehlere) [reply pv_vardict_0.0.1/single_filter_0.0.1.cwl]: fix in this commit: https://github.com/msk-access/cwl-commandlinetools/pull/148/commits/be5e53e5c5b8838222b3b1f477a0785684f4dab0 !

**Comment by buehlere** [comment]
> We might want to call the folder `pv` and then add version `0.0.1` and then the files can be `pv_vardict_filter_case-control.cwl` and `pv_vardict_filter_single-sample.cwl`. Also please add the folder pv with `README.md` with just `# Posprocessing of variant calls` and `vardict_filter_case-control.md` and `vardict_filter_single-sample.md`

@rhshah I think this pull should address most of the re-structuring: https://github.com/msk-access/cwl-commandlinetools/pull/148/commits/63dd20b0fc39935d1fd39cbe9fa0ced233c755c6 !

**Comment by buehlere** [comment]
Documentation is updated in this commit: https://github.com/msk-access/cwl-commandlinetools/pull/148/commits/6d6ef17217f491d77f004eb4c6f3acd2b7e190b0

**Pull request merged**